src/SuperVault.sol

```diff
@@ -274,18 +283,18 @@ contract SuperVault is BaseStrategy, ISuperVault {
 
     /// @inheritdoc ISuperVault
     function setWhitelist(
-        uint256[] memory superformIds,
-        bool[] memory isWhitelisted
+        uint256[] memory superformIds_,
```

Comment:
this can do add/remove on the set

Reply:
The function already performs this via _changeSuperformWhitelist